src/service_manager.py

```diff
@@ -157,13 +157,24 @@ async def _create_view(
         store: ArangoStorage,
         data_product: str,
         db_collection_name: str,
+        create_generic_view: bool = False
     ):
-    view_spec = load_specs.load_spec(data_product)
-    print("Found view specs:")
-    for v in sorted(view_spec.spec_files):
-        print(f"   {v}")
+    if create_generic_view:
+        print(f"Creating generic view for data product {data_product}...")
+        view_spec = generic_view.create_generic_spec()
+        # TODO: force creating/replacing existing generic view
+    else:
+        print(f"Loading view spec for data product {data_product}...")
+        view_spec = load_specs.load_spec(data_product)
+        print("Found view specs:")
+        for v in sorted(view_spec.spec_files):
+            print(f"   {v}")
+
     views = sorted(await store.get_search_views_from_spec(
-        db_collection_name, view_spec, analyzers.get_analyzer))
+        db_collection_name,
+        view_spec,
+        analyzers.get_analyzer,
+        include_all_fields=create_generic_view))
     _print_view_status(data_product, db_collection_name, views)
     if views:
         print("    Setup ok.")
```

Comment:
Below the generic view is assigned a view name by the user. How is the service going to know which view to use?

Reply:
it should be specified at `save collection` step. Same as genome attributes.

Reply:
But why? it doesn't depend on the a manual spec change so we can do this automatically. There's no reason to put the user in the loop at all

Reply:
hmm once the view is created. I don't see why we are doing differently between genome attributes and the heatmap. Maybe we should just always auto link newly created view to the corresponding collection?

Reply:
I'm saying there's no reason to link a generic view to a particular collection. It's generic, it applies to all versions of all collections

Reply:
👍 creating a common name for generic view. I also updated `get_data_product` in models.py. If you think we should instead get product exactly as what's stored in DB. I can revert that and get the common search view name when I work on updating FilterSet in heatmap.py.

Reply:
:+1: